projects/natural/src/lib/services/theme.service.ts

```diff
@@ -0,0 +1,84 @@
+import {
+    DOCUMENT,
+    EnvironmentProviders,
+    inject,
+    Injectable,
+    InjectionToken,
+    provideAppInitializer,
+    Provider,
+    signal,
+} from '@angular/core';
+import {LOCAL_STORAGE} from '../modules/common/services/memory-storage';
+
+export const NATURAL_THEMES_CONFIG = new InjectionToken<string[]>('Configuration for Natural Theme');
+
+export function provideThemes(config: string[]): (EnvironmentProviders | Provider)[] {
```

Comment:
Force de déclarer au minimum un thème via le typing, plutôt que via runtime:

```suggestion
type AllThemes = [string, ...string[]];
export const NATURAL_THEMES_CONFIG = new InjectionToken<AllThemes>('Configuration for Natural Theme');

export function provideThemes(config: AllThemes): (EnvironmentProviders | Provider)[] {
```